### DISCUSSION TITLE
Plugin 'Gauge' (version '242.19890.14') is not compatible with the current version of the IDE, because it requires build 242.19890 or newer but the current build is QA-241.17011.162

### DISCUSSION THREAD

**Discussion by antondebeer** [getgauge/gauge#2567] [General]
I am trying to install the latest gauge plugin, from disk because on IntelliJ plugin manager it shows I am up to date with version 241.14494.158 but the latest is 242.19890.14, but when I try to install it I get the error: Plugin 'Gauge' (version '242.19890.14') is not compatible with the current version of the IDE, because it requires build 242.19890 or newer but the current build is QA-241.17011.162.
How can I get the latest gauge plugin on my IntelliJ project

**Comment by zabil** [comment]
Please try raising this issue with the IntelliJ team at https://youtrack.jetbrains.com/issues/IDEA
